bldsys/cmake/module/FindVulkan.cmake

```diff
@@ -329,7 +329,7 @@ find_library(Vulkan_LIBRARY
         HINTS
         ${_Vulkan_hint_library_search_paths}
 )
-message(STATUS "vulkan_library ${Vulkan_LIBRARY} search paths ${_Vulkan_hint_library_search_paths}")
+#message(STATUS "vulkan_library ${Vulkan_LIBRARY} search paths ${_Vulkan_hint_library_search_paths}")
```

Comment:
Any reason this was commented out?

Reply:
Yes, it seems this is a debug message, and provides almost the same information as when FindVulkan prints out the found Vulkan library.  A minor change, but a bit of noise reduction in the log.  If you want it, I can always revert.

Reply:
It's fair enough to just delete this.  It was helpful when I wrote this the first time and it does prove helpful from time to time when things get moved around locally; but reduction in noise is a good thing.

Reply:
@gpx1000 thanks for the background.  I left it commented-out but still present in the code in case you want to reenable it for debugging, etc.